be/src/storage/lake/general_tablet_writer.h

```diff
@@ -14,12 +14,14 @@
 
 #pragma once
 
+#include <future>
 #include <memory>
 #include <string>
 #include <vector>
 
 #include "gutil/macros.h"
 #include "storage/lake/tablet_writer.h"
+#include "util/threadpool.h"
```

Comment:
We can remove this include and use forward declartion of `ThreadPool` to reduce compile time.